sklearn/metrics/_plot/roc_curve.py

```diff
@@ -374,11 +374,10 @@ def plot_roc_curve(
 
     Read more in the :ref:`User Guide <visualizations>`.
 
-    .. deprecated:: 1.0
-      `plot_roc_curve` is deprecated in 1.0 and will be removed in
-       1.2. Use one of the following class methods:
-       :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` or
-       :func:`~sklearn.metrics.RocCurveDisplay.from_estimator`.
+    `plot_roc_curve` is deprecated in 1.0 and will be removed in
```

Comment:
OK, I just saw the rendering. We could remove this part indeed and change the deprecated above:

```
@deprecated(
    "Function :func:`plot_roc_curve` is deprecated in 1.0 and will be "
    "removed in 1.2. Use one of the class methods: "
    ":meth:`sklearn.metric.RocCurveDisplay.from_predictions` or "
    ":meth:`sklearn.metric.RocCurveDisplay.from_estimator`."
)
```